packages/web-pkg/src/helpers/resource/conflictHandling/conflictDialog.ts

```diff
@@ -27,7 +27,8 @@ export class ConflictDialog {
         [key: string]: string
       },
       disableHtmlEscaping?: boolean
-    ) => string
+    ) => string,
+    protected isUpload?: boolean
```

Comment:
Left-over from the previous approach I believe?

Reply:
Exactly